components/docsMain/docsMobileHeader.tsx

```diff
@@ -0,0 +1,77 @@
+import { LeftHandSideHeader } from 'components/docsSearch/SearchNavigation';
+import { useState } from 'react';
+import { FaChevronRight } from 'react-icons/fa';
+import DirectoryOverflowButton from './directoryOverflowButton';
+
+export const MobileVersionSelect = () => {
+  const versions = [
+    ['v.Latest', 'https://tina.io'],
```

Comment:
could change the link the just ' ' since we negate the movement anyway in `handleVersionClick()`

Reply:
done